src/main/com/example015/radek/RenameFilesApplication.java

```diff
@@ -0,0 +1,12 @@
+package main.com.example015.radek;

```

Comment:
Jsem mile překvapen.

Pofixuj jen highlighted věci od IDEI ... plus to spouštění.

Reply:
Mám radost :) 

Spouštění oddělené z konstruktoru a chyby fixnuté

Reply:
No ale mám tam jeden problém ... to vytáhne ty přejmenované fily o složku výš. To jsme rozhodně nechtěli.

Reply:
Už by to mělo fungovat, problém je, že linux používá "/", ale windows "\" takže, jsem použil file.separator a teď už by se měl používat dle potřeby ten správný